### DISCUSSION TITLE
Closing your own health interface makes you grab cursor position character

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14126] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/FakeFishGames/Barotrauma/assets/64390906/41b3c222-9015-40a3-baa0-ae767731ac92

in the video I was using deselect key to close the interface, but it still made my character grab others
the issue only exists in multiplayer

### Reproduction steps

1. start a multiplayer
2. go to find someone else
3. open your own health interface
4. move your cursor on others
5. close your health interface
6. automatic grabbing

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Vykn** [comment]
Happens a lot. Very awkward when your crewmates asks why you're grabbing them and you didn't even realise it.

**Comment by A2ur3** [comment]
the issue still exists in the latest version

**Comment by Regalis11** [comment]
Thank you for the report! This has now been addressed in our private repo, fix coming up in the next unstable build and an upcoming release.
